front/src/components/UI/Header/HeaderStyle.css

```diff
@@ -0,0 +1,58 @@
+.main-panel{
```

Comment:
Przykład jak zmigrować style CSS do Styled Components na przykładzie tej klasy:
`
const MainPanel = styled.div'
  color: white;
  width: 100%;
  height: 60px;
'
` - tylko zamiast pojedynczego apostrofa użyj tego backsticka który jest na klawiszu z tyldą 

Poźniej w pliku .tsx jedynie wykorzystujesz to jak zwyczajny komponent Reactowy czyli robisz 
`<MainPanel>jakiś content</MainPanel>`

Reply:
Gotowy przykład jest zrobiony w folderze Button i tam jest plik tsx oraz styled.ts